pkg/handler/processor/guesser/type_guesser.go

```diff
@@ -38,7 +39,7 @@ var (
 
 func RegisterDocumentTypeGuesser(g DocumentTypeGuesser, name string) {
 	if _, ok := documentTypeGuessers[name]; ok {
-		logrus.Warnf("the document type guesser is being overwritten: %s", name)
+		fmt.Printf("the document type guesser is being overwritten: %s", name)
```

Comment:
all `Register*` functions should probably have the same pattern, we should stick to one - i don't this we should have warning messages print to stdout using `fmt`.. I think returning an err may be a good option here (see comment below).

Reply:
Addresed ✅